posts/squidpy-essentials/squidpy-essentials.qmd

```diff
@@ -0,0 +1,1071 @@
+---
+title: "Using Squidpy with AtoMx&#8482; SIP exports"
+author:
+  - name: Evelyn Metzger
+    orcid: 0000-0002-4074-9003
+    affiliations: 
+      - ref: nstg
+      - ref: eveilyeverafter
+toc: true
+toc-title: Contents
+toc-depth: 3
+toc-expand: 2
+toc-location: left
+number-sections: true
+number-depth: 3
+date: "2024-06-24"
+categories: [squidpy]
```

Comment:
anndata, python, visualization, how-tos are other categories we've used before that may make sense to include

Reply:
Done: 2ee3299f7b19ca4c6e5e91a5d32d1428c4cab7bd